App/app/src/main/java/com/wokabel/app/wokabel/models/Vocable.java

```diff
@@ -1,29 +1,50 @@
-package com.wokabel.app.wokabel.Models;
+package com.wokabel.app.wokabel.models;
+
+import android.arch.persistence.room.ColumnInfo;
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.Ignore;
+import android.arch.persistence.room.PrimaryKey;
+import android.support.annotation.NonNull;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.UUID;
 
+
+
 /**
  * A vocable
  */
+@Entity(tableName = "vocablist")
 public class Vocable {
+    @PrimaryKey
+    @NonNull
     private String key;
-    private ArrayList<String> values;
+
+    private String values;
```

Comment:
Hier wäre es trotzdem besser die `values` noch in ner ArrayList zu haben.
Also so:
    private String values; // Fuer Datenbank
    @Ignore
    private ArrayList valuesList; // Fuer den vereinfachten Umgang mit dem Objekt

Reply:
Oder gäbe es dann irgendwelche Probleme mit der Synchronisation zwischen Objekt und Datenbank?

Reply:
Wenn ja dann wenigstens eine Methode in `Vocable` die dir die `values` wieder zu einer `ArrayList<String>` konvertiert und zurückgibt.

Reply:
**Ich glaube das mit der Methode ist sogar am besten.**

Reply:
Datenbanken können nicht mit ArrayLists arbeiten, wenn du das meinst

Reply:
Ja ich weiß aber wir können nicht mit diesem generierten String ("Hey, Ho, Hallo") arbeiten. Deswegen brauchen wir eine Schnittstelle dazwischen. z.B. Die Methode die den String konvertiert.

Reply:
ist doch schon drinnen ?! 
`public ArrayList<String> getValuesList()`
Ich brauche die normalen getter und Setter für Room. Innerhalb von ROOM wird es in Form von einem String bearbeitet. Außerhalb als ArrayList.

Reply:
Oops mein Fehler.

Dann kannst mergen!